src/loader.js

```diff
@@ -244,7 +244,8 @@ module.exports = function loader(content, map, meta) {
     await Promise.all(dependencies);
 
     styles.forEach(style => {
-      emitVirtualFile(style.absoluteFilePath, style.value);
+      const mtime = emitVirtualFile(style.absoluteFilePath, style.value);
+      compilation.fileTimestamps.set(style.absoluteFilePath, +mtime);
```

Comment:
Would you mind linking this PR in a comment here for future context on why this is required?

Reply:
Sure thing, I'm on BST at the moment, so it's early hours (and away from my machine) if your keen to land it ahead of any timezone cycle, I'm not adverse to you adding the link on my behalf.

Reply:
All done.